client/my-sites/plans-features-main/plan-type-selector.tsx

```diff
@@ -382,6 +373,20 @@ const StyledPopover = styled( Popover )`
 			}
 		}
 
+		&.is-bottom-right,
```

Comment:
I'd ideally like to not add the styles for `is-bottom-right` but these styles fixed a flicker issue I saw on scroll. Check below on the flicker that is seen without the `is-bottom-right` styles:




![mnthlydef](https://user-images.githubusercontent.com/1269602/148541035-414ba408-8bdc-42d3-a7a5-92255824346e.gif)